competing-consumers/CompetingConsumers/CompetingReceiverApp/Program.cs

```diff
@@ -0,0 +1,77 @@
+﻿namespace ReceiverApp
+{
+ using Microsoft.Azure.ServiceBus;
+ using System;
+ using System.Configuration;
+ using System.Text;
+ using System.Threading;
+ using System.Threading.Tasks;
+
+ class Program
+ {
+  // Connection String for the namespace can be obtained from the Azure portal under the 
+  // 'Shared Access policies' section.
+  private static readonly string queueName = ConfigurationManager.AppSettings.Get("QueueName");
+  private static readonly string serviceBusConnectionString = ConfigurationManager.AppSettings.Get("ServiceBusConnectionString");
+  static IQueueClient queueClient;
+  private static bool keepRunning = true;
+
+  static void Main(string[] args)
+  {
+   AppDomain.CurrentDomain.ProcessExit += new EventHandler(CurrentDomain_ProcessExit);
+
+   MainAsync().GetAwaiter().GetResult();
```

Comment:
Can we use `HostBuilder` and `AddServiceBus()` instead of this hand-rolled solution?  That would then be up to date with how we actually expect a consumer (receiver) to be generally written by a customer. (might save you a few lines of code as well)